cmake/modules/RootMacros.cmake

```diff
@@ -370,29 +470,38 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
   endif()
 
   if(CMAKE_PROJECT_NAME STREQUAL ROOT)
-    set(includedirs -I${CMAKE_SOURCE_DIR}
-                    -I${CMAKE_BINARY_DIR}/etc/cling/ # This is for the RuntimeUniverse
-                    -I${CMAKE_BINARY_DIR}/include)
+    list(APPEND includedirs -I${CMAKE_BINARY_DIR}/include)
+    list(APPEND includedirs -I${CMAKE_BINARY_DIR}/etc/cling) # This is for the RuntimeUniverse
+#    list(APPEND includedirs -I${CMAKE_SOURCE_DIR})
     set(excludepaths ${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
   elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/inc)
     set(includedirs -I${CMAKE_CURRENT_SOURCE_DIR}/inc)
   endif()
   foreach( d ${incdirs})
-   set(includedirs ${includedirs} -I${d})
+    list(APPEND includedirs -I${d})
   endforeach()
 
   foreach(dep ${ARG_DEPENDENCIES})
     if(TARGET ${dep})
       get_property(dep_include_dirs TARGET ${dep} PROPERTY INCLUDE_DIRECTORIES)
       foreach(d ${dep_include_dirs})
-        set(includedirs ${includedirs} -I${d})
+         list(APPEND includedirs -I${d})
       endforeach()
     endif()
   endforeach()
-
+  
```

Comment:
Stray spaces

Reply:
I switch off all re-formatting having lot of complains in past.
Lot of codes in roofit or tmva has such problems and by saving I effectively touching 10% of lines